src/app/pages/webinars/webinars.js

```diff
@@ -28,8 +28,9 @@ function Webinars({data: {heading: headline, description, heroImage}}) {
     const index = firstFuture > 0 ? firstFuture : webinarData.length;
     const upcomingData = webinarData.slice(index);
     const pastData = webinarData.slice(0, index);
+    const emptyMessage = webinarData.noWebinarsMessage || '(No webinars)';
```

Comment:
Am I looking at this right, the array will get this as a property? That feels unintuitive but maybe makes sense with how the JSON data is structured?

Reply:
If the JSON is easy to modify, this might feel better as an object:
```javascript
{ emptyMessage: '(No webinars)', events: [...the current webinar data] }
```

Reply:
The empty message will be pulled separately, so I moved it closer to where it's used.